examples/reference/elements/matplotlib/Bars.ipynb

```diff
@@ -73,15 +73,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Bars.Grouped [category_index=0 group_index=1 stack_index=5] \n",
-    "%%opts Bars.Stacked [color_by=['stack'] stack_index=1 category_index=5]\n",
     "from itertools import product\n",
     "np.random.seed(3)\n",
     "index, groups = ['A', 'B'], ['a', 'b']\n",
     "keys = product(index, groups)\n",
     "bars = hv.Bars([k+(np.random.rand()*100.,) for k in keys],\n",
     "               ['Index', 'Group'], 'Count')\n",
-    "bars.relabel(group='Grouped') + bars.relabel(group='Stacked')"
+    "\n",
+    "grouped = bars.relabel(group='Grouped')\n",
+    "stacked = bars.relabel(group='Stacked')\n",
+    "\n",
+    "grouped.opts(stack_index=5) + stacked.opts(stack_index=1, category_index=5, color_by=['stack'])"
```

Comment:
This example is awkward since we're trying to style the two Bars separately, I would switch to use ``opts`` on the ``stacked`` bars here but it's really ugly.